app/jobs/fix_pubmed_entries_missing_authors.rb

```diff
@@ -1,11 +1,19 @@
 class FixPubmedEntriesMissingAuthors < ActiveJob::Base
+
+  attr_reader :recurring
+
+  after_perform do |job|
```

Comment:
Should we make a parent class that pulls out the after_perfom?

Reply:
(just re-read your pull request description and I agree that we should add a mixin or parent class to pull out the after_perfom)